client/src/routes/RootRouter.tsx

```diff
@@ -13,7 +13,7 @@ const RootRoutes: React.FC = () => {
 
     return (
         <Layout>
-            <Header style={{ position: 'fixed', zIndex: 1, width: '100%', backgroundColor: 'transparent', padding: '0', top: 0}}>
+            <Header style={{ position: 'fixed', zIndex: 10, width: '100%', backgroundColor: 'transparent', padding: '0', top: 0}}>
```

Comment:
We should put styles in css files. We shouldn't use inline style. Also please try to avoid using z-index as much as possible. See [this](https://danielrotter.at/2020/04/08/avoid-z-index-whenever-possible.html) for more information